.gitignore

```diff
@@ -13,4 +13,5 @@ wallaby.conf.js
 .DS_Store
 src/types/coverage/
 .npmrc
+tv-automation-state-timeline-resolver
```

Comment:
Not sure how this is relevant?

Reply:
I agree, it should probably not be in here.

It comes from https://github.com/olzzon/tv-automation-state-timeline-resolver/commit/016065b2dfb782c58f1976c428c213fc8ffabbce

@thomasslee97 Is this a remnant from a local file structure perhaps?

Reply:
This shouldn't be here for NRK